server/backend/pytest.ini

```diff
@@ -15,6 +15,6 @@ markers =
 
 env =
     COGNITO_REGION=ca-central-1
-    COGNITO_USER_POOL_ID=ca-central-1_yds9Vci8g
-    COGNITO_CLIENT_ID=75g60i0g3rt0npuvvho1ucvjm8
+    COGNITO_USER_POOL_ID=ca-central-1_SMbUfdCC3
+    COGNITO_CLIENT_ID=7oi2u0st9ov66aahdk02at9f7g
```

Comment:
just put a comment here, looks like the cognito credentials are only needed for sqlite testing, we could consider remove them once we rewrite all the tests using test container